htdocs/public/partnership/new.php

```diff
@@ -211,7 +216,8 @@ function llxFooterVierge()
 			$partnership->fk_soc = 0;
 		}
 
-		$partnership->statut      = -1;
+		// need to change this part
+		$partnership->status      = 0;
 		$partnership->firstname   = GETPOST('firstname');
```

Comment:
firstname and lastname must be commented. They are not field of $partnership.
Add them into field    $company->name_alias = dolGetFirstnameLastname(firstname, lastname);